### ISSUE TITLE
Blame info not working

### ISSUE THREAD

**Issue by jimis** [Ericsson/codechecker#3443]
I upgraded to v6.17.0 and I don't see the new blame info button activated. In the logs I see:

```
+ CodeChecker store /home/user/codechecker_results/qtbase-dev-20211008-1bac82fa99 --trim-path-prefix /home/user/src/qt/qt5/qtbase/ -n qtbase-dev-20211008-1bac82fa99 --url http://localhost:8002/qtbase-dev
[INFO 2021-10-08 02:28] - Storing analysis results for run 'qtbase-dev-20211008-1bac82fa99'
[INFO 2021-10-08 02:28] - Checking for local valid sessions.
[INFO 2021-10-08 02:28] - Processing report files done.
[INFO 2021-10-08 02:28] - Collecting review comments ...
[INFO 2021-10-08 02:28] - Collecting review comments done.
[INFO 2021-10-08 02:28] - Get missing file content hashes from the server...
[INFO 2021-10-08 02:28] - Get missing file content hashes done.
[INFO 2021-10-08 02:28] - Collecting blame information for source files...
/usr/lib/python3.8/zipfile.py:1517: UserWarning: Duplicate name: 'root/home/user/src/qt/qt5/qtbase/src/corelib/kernel/qobject_p.h'
  return self._open_to_write(zinfo, force_zip64=force_zip64)
/usr/lib/python3.8/zipfile.py:1517: UserWarning: Duplicate name: 'root/home/user/src/qt/qt5/qtbase/src/gui/platform/unix/dbusmenu/qdbusmenutypes_p.h'
  return self._open_to_write(zinfo, force_zip64=force_zip64)
/usr/lib/python3.8/zipfile.py:1517: UserWarning: Duplicate name: 'root/home/user/src/qt/qt5/qtbase/src/corelib/text/qchar.h'
  return self._open_to_write(zinfo, force_zip64=force_zip64)
/usr/lib/python3.8/zipfile.py:1517: UserWarning: Duplicate name: 'root/home/user/src/qt/qt5/qtbase/src/corelib/kernel/qobject.h'
  return self._open_to_write(zinfo, force_zip64=force_zip64)
/usr/lib/python3.8/zipfile.py:1517: UserWarning: Duplicate name: 'root/home/user/src/qt/qt5/qtbase/src/gui/rhi/qrhi_p.h'
  return self._open_to_write(zinfo, force_zip64=force_zip64)
/usr/lib/python3.8/zipfile.py:1517: UserWarning: Duplicate name: 'root/home/user/src/qt/qt5/qtbase/src/corelib/global/qsimd_p.h'
  return self._open_to_write(zinfo, force_zip64=force_zip64)
/usr/lib/python3.8/zipfile.py:1517: UserWarning: Duplicate name: 'root/home/user/src/qt/qt5/qtbase/src/corelib/itemmodels/qitemselectionmodel.h'
  return self._open_to_write(zinfo, force_zip64=force_zip64)
/usr/lib/python3.8/zipfile.py:1517: UserWarning: Duplicate name: 'root/home/user/src/qt/qt5/qtbase/src/corelib/io/qfile.h'
  return self._open_to_write(zinfo, force_zip64=force_zip64)
[INFO 2021-10-08 02:29] - Collecting blame information done.
[INFO 2021-10-08 02:29] - Storing results (10.9MiB) to the server...
[INFO 2021-10-08 02:29] - Storage finished successfully.
```

I am assuming it's related to using `--trim-path-prefix`.
Also related to the fact that some paths are prepended with `'root'`, because of 
https://github.com/Ericsson/codechecker/blob/0168f93676eec7711d4066da6d990ea0e5aa8e75/web/client/codechecker_client/cmd/store.py#L688

I believe that every file is not even found in `__get_blame_info()` and that returns silently:

https://github.com/Ericsson/codechecker/blob/0168f93676eec7711d4066da6d990ea0e5aa8e75/web/client/codechecker_client/blame_info.py#L19-L24

**Comment by csordasmarton** [comment]
There is a known issue in our side with the blame information storage that we do not store blame information for source files which are already stored into the database. We will try to fix it in the next release to store blame information for every source files which were stored previously to the database (with an old version of CodeChecker which doesn't support blame information storage) and do not have blame information.

Can you please try to store your results to a new product or to a new database to make sure that this will solve the problem? If not then it will be a different problem.

**Comment by jimis** [comment]
Indeed, the same results are being stored with correct blame info into a new product.
I got confused by these zipfile.py warnings with the "root" prefix. I wonder why I get those.

**Comment by bruntib** [comment]
Solved by #3488.
